static/admin/config.yml

```diff
@@ -8,22 +8,20 @@ publish_mode: editorial_workflow
 media_folder: "src/images" # where media files will be stored in the repo
 
 collections:
-  - name: "blog" # Used in routes, e.g., /admin/collections/blog
-    label: "Blog" # Used in the UI
-    folder: "src/pages/blog" # The path to the folder where the documents are stored
-    create: true # Allow users to create new documents in this collection
-    slug: "{{year}}-{{month}}-{{day}}-{{slug}}" # Filename template, e.g., YYYY-MM-DD-title.md
-    fields: # The fields for each document, usually in front matter
-      - {label: "Layout", name: "layout", widget: "hidden", default: "blog"}
-      - {label: "Path", name: "path", widget: "string"}
-      - {label: "Date", name: "date", widget: "datetime"}
-      - {label: "Title", name: "title", widget: "string"}
-      - {label: "Body", name: "body", widget: "markdown"}
   - name: "project"
     label: "Project"
     folder: "src/pages/project"
     create: "true"
     slug: "{{title}}"
     fields:
       - {label: "Title", name: "title", widget: "string"}
-      - {label: "Body", name: "body", widget: "markdown"}
+      - {label: "Summary Description", name: "summary", widget: "markdown"}
+      - {label: "Stage", name: "stage", widget: "select", options: ["Proposed", "Planning", "Implementing", "Live"]}
+      - {label: "Budget", name: "budget", widget: "number", required: false}
+      - {label: "Funding Source", name: "funding_source", widget: "string", required: false}
+      - {label: "Lead Department", name: "department", widget: "string", required: false}
+      - {label: "Partner Organizations", name: "partners", widget: "string", required: false}
+      - {label: "Contact Person", name: "contact_name", widget: "string", required: false}
+      - {label: "Contact Email", name: "contact_email", widget: "string", required: false}
+      - {label: "Related City Goals", name: "goals", widget: "string", required: false}
+      - {label: "Collection Type", name: "type", widget: "hidden", default: "project"}
```

Comment:
It was useful to create a hidden field so that I could have a consistent thing to filter one to display the projects in a list on the landing page. But I was surprised there wasn't already some Collection Type available. This seems like exactly the type of thing the hidden widget was made for but also feels hacky.